curriculum/challenges/_meta/responsive-web-design-projects/meta.json

```
@@ -1,6 +1,7 @@
 {
   "name": "Responsive Web Design Projects",
-  "isUpcomingChange": false,
+  "isUpcomingChange": true,
```

```suggestion
  "isUpcomingChange": false,
```
This is what's got Cypress upset - with this set to true, this block doesn't get its pages built.

---

🤦